taxi/models.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(fields=["name"], name="unique_name")
+        ]
```

Comment:
it's easier to use unique=True inside field name, but then you want some fields to be unique together(unique_together in Meta), then yes, it's better to use unique constraint